rtv/content.py

```diff
@@ -70,9 +72,16 @@ def strip_praw_comment(comment):
             data['body'] = comment.body
             data['created'] = humanize_timestamp(comment.created_utc)
             data['score'] = '{} pts'.format(comment.score)
-            data['author'] = (comment.author.name if getattr(comment, 'author') else '[deleted]')
-            data['is_author'] = (data['author'] == getattr(comment.submission, 'author'))
-            data['flair'] = (comment.author_flair_text if comment.author_flair_text else '')
+            data['author'] = (
+                comment.author.name if getattr(
+                    comment,
+                    'author') else '[deleted]')
+            data['is_author'] = (
+                data['author'] == getattr(
+                    comment.submission,
+                    'author'))
+            data['flair'] = (
+                comment.author_flair_text if comment.author_flair_text else '')
```

Comment:
These look totally mangled. If the only violation here is 80 lines, I would rather break it for the sake of readability. Or get creative with it, e.g.
```python
c_author = getattr(comment, 'author')
data['author'] = comment.author.name if c_author else '[deleted']
```